### PR TITLE
Docs/api key creation guide

### LINKED ISSUE #221: Add documentation for how to set up the Google Sheets API key
Add documentation to the CONTRIBUTING.md about how to get the google sheets API key to work locally.

### PR DESCRIPTION
**Pull request by ChrisChrisLoLo**
Add documentation about setting up the Google Sheets API key locally in development
- Add template file
- Add instructions to `CONTRIBUTING.md`
- Rename environmental variable names to be consistent with the ones in the `.env` file
- Have Prettier make style changes

Resolves #221

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -32,8 +32,8 @@ export const appConfiguration = {
   },
   plugins: [
     replace({
-      "process.env.G_CLIENT_ID": JSON.stringify(process.env.CLIENT_ID),
-      "process.env.G_API_KEY": JSON.stringify(process.env.API_KEY),
+      "process.env.CLIENT_ID": JSON.stringify(process.env.CLIENT_ID),
+      "process.env.API_KEY": JSON.stringify(process.env.API_KEY),
```
Comment (yehee) [inline rollup.config.js]:
```suggestion
      CLIENT_ID: JSON.stringify(process.env.CLIENT_ID),
      API_KEY: JSON.stringify(process.env.API_KEY),
```
This way we don't have to reassign it later on `GoogleSheetsInput.svelte`.

File: src/app/components/GoogleSheetsInput.svelte [hunk 2]
```diff
@@ -7,8 +7,8 @@
   let googleSheetsURL = "";
 
   // Client ID and API key from the Developer Console
-  const CLIENT_ID = process.env.G_CLIENT_ID;
-  const API_KEY = process.env.G_API_KEY;
+  const CLIENT_ID = process.env.CLIENT_ID;
+  const API_KEY = process.env.API_KEY;
```
Comment (yehee) [inline src/app/components/GoogleSheetsInput.svelte]: Shouldn't be necessary if the above change is in place!

**Review by yehee — CHANGES_REQUESTED** [review]
Looks great! Thanks for putting this all together!

File: rollup.config.js (hunk 1 above)
Comment (ChrisChrisLoLo) [reply rollup.config.js]: Works for me! I'll let you merge #228 and then I'll make my PR conform accordingly

**Review by yehee — APPROVED** [review]
Looks good to me! :shipit:

**Pull request merged**